polkadot/xcm/src/lib.rs

```diff
@@ -260,52 +418,72 @@ versioned_type! {
 	pub enum VersionedJunction {
 		#[codec(index = 2)]
 		V2(v2::Junction),
-		#[codec(index = 3)]
+		#[codec(index = 3)] // Same as v4
 		V3(v3::Junction),
```

Comment:
may be `V4(v3::Junction)` ?

Reply:
I actually have a v4::Junction, will use that instead